CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -118,7 +118,7 @@ function Staff:tick()
   -- check if we need to use the staff room and go there if so
   self:checkIfNeedRest()
   -- check if staff has been waiting too long for a raise and fire if so
```

Comment:
Not you, but this comment is inaccurate (not expecting you to address it)

Reply:
> Not you, but this comment is inaccurate (not expecting you to address it)

Agree. Fixed.